Modules/_testcapimodule.c

```diff
@@ -5430,6 +5430,38 @@ test_refcount(PyObject *self, PyObject *Py_UNUSED(ignored))
 }
 
 
+// Test Py_Is(), Py_IsNone(), Py_IsTrue() and Py_IsFalse() functions
+static PyObject*
+test_py_is(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    PyObject *o_none = Py_None;
+    PyObject *o_true = Py_True;
+    PyObject *o_false = Py_False;
+    PyObject *obj = PyList_New(0);
+
```

Comment:
Check return value :)

```suggestion
    if (obj == NULL) {
        return NULL;
    }

```

Reply:
Oh thanks, fixed.